src/kernel/bitmap.zig

```diff
@@ -89,19 +89,21 @@ pub fn ComptimeBitmap(comptime BitmapType: type) type {
         /// Arguments:
         ///     IN/OUT self: *Self - The bitmap to modify.
         ///     IN num: usize - The number of entries to set.
+        ///     IN from: ?IndexType - The entry number to start allocate from or null if it can start anywhere
         ///
         /// Return: ?IndexType
         ///     The first entry set or null if there weren't enough contiguous entries.
+        ///     If `from` was not null and any entry between `from` and `from` + num is set then null is returned.
         ///
-        pub fn setContiguous(self: *Self, num: usize) ?IndexType {
+        pub fn setContiguous(self: *Self, num: usize, from: ?IndexType) ?IndexType {
```

Comment:
From the previous comments, you said that `from` is the entry to start from and not the bit index, so this should be `?usize`

Reply:
But this is in the comptime bitmap where an entry in the bitmap is denoted with an IndexType.

Comptime bitmap: ++++++++++++++++++++++++++++++++++++++++
_______________|-----------------^--------------------------------------------------|
                                              IndexType

Normal bitmap: [++++++++][++++++++][++++++++][++++++++][++++++++]
IndexType: _____________|----^--------|
usize: _______|---------------------^--------------------------------------------------------|

An IndexType in the normal bitmap refers to an index within one of the array elements but refers to an index within the whole bitmap in the comptime one (underscores are just for padding).